siliconcompiler/tools/vpr/__init__.py

```diff
@@ -66,6 +66,12 @@ def __init__(self):
         self.define_tool_parameter("vpr", "clock_model", "<ideal,route,dedicated_network>",
                                    "The clock modeling strategy to be used.")
 
+        self.define_tool_parameter("vpr",
```

Comment:
mark this a required in the setup method.

Reply:
Ohh, I see, is this a thing the FPGA should control? It seems like this should be a parameter for the task driver